src/app/header/header.component.html

```diff
@@ -63,7 +63,10 @@
         </ul>
       </li>
       <li routerLinkActive="active">
-        <a [routerLink]="['/work-item-list']" id="header_menuWork">Work</a>
+        <a [routerLink]="['/work-item/list']" id="header_menuWork">WorkList</a>
```

Comment:
Actually this section is not our section. It's just for a URL navigation. This header section will be a part of platform UI.